auth/authn-authz-intro.md

```diff
@@ -0,0 +1,37 @@
+# Modernising Authentication and Authorization for Network Devices
+**Contributors:** danielwong<sup><small>†</small></sup>, johnwestbrook<sup><small>†</small></sup>, robjs<sup><small>†</small></sup>, morrowc<sup><small>†</small></sup>, aashaikh<sup><small>†</small></sup>  
+<sup><small>†</small></sup> @google.com  
+**Published:** November 2020
+
+## Problem Statement
+Historically, network devices have had a single store of configuration data, users with sufficient privilege levels are allowed to edit this data. Implementation-specific mechanisms have been used to restrict the subsets of configuration and telemetry that a particular user can interact with, such that it is possible to enforce particular roles interacting with a network device in specific manners. Often, the authorization policies that network operators have implemented have been coarse - for example, high privilege users being able to edit any configuration, or a single SNMP community having full access to all telemetry on the device. Typically, only lawful intercept credentials and configuration has been subject to a 'protected' configuration store, which was treated differently by the device.
+
+The OpenConfig project initially provided means to inter-work with existing means of authorization and authentication on network devices - providing means to supply a username and password via gRPC metadata for gNMI and gNOI - along with mutual TLS authentication. However, this approach has a number of limitations:
```

Comment:
Re. "mutual TLS authentication", the wording in the current [gNMI Authentication and Encryption](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-authentication.md#requirements-on-the-network-device-gnmi-target) seems to imply server authentication only.

Reply:
ACK - I think this originally was written with the context being that it was being sent to vendors. Is the clarification that we need to make here that we expect that the client also authenticates the server's CN based on its expectation. Is it CN that we're proposing to use here, or does the device itself need a cert with an associated SPIFFE ID?